src/components/Calendar/EventModal.tsx

```diff
@@ -43,9 +50,11 @@ const EventModal: React.FC<{
             isOpen={true}
```

Comment:
Probably unrelated but a conditionally rendered modal isn't great for accessibility.

Reply:
Will fix it on VFB-2